<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KartePoIAnlegen</class>
 <widget class="QDialog" name="KartePoIAnlegen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Bestaetigen">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bestaetigen</string>
   </property>
  </widget>
  <widget class="QLabel" name="Laengengrad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Laengengrad</string>
   </property>
  </widget>
  <widget class="QLabel" name="Breitengrad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Breitengrad</string>
   </property>
  </widget>
  <widget class="QLabel" name="Kategorie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kategorie</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bemerkung">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bemerkung</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LaengengradLine">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BreitengradLine">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="KategorieLine">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BemerkungLine">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
